sdk/storage/tests.yml

```diff
@@ -23,3 +23,8 @@ jobs:
         AZURE_STORAGE_QUEUE_ENDPOINT: $(java-storage-test-queue-endpoint)
         AZURE_STORAGE_QUEUE_CONNECTION_STRING: $(java-storage-test-queue-connection-string)
         AZURE_STORAGE_BLOB_CONNECTION_STRING: $(java-storage-test-blob-connection-string)
+      PreRunSteps:
```

Comment:
It looks like we're not setting any environment variables in this change that will point the tests to Azurite.

Reply:
Azurite uses a publicly documented endpoint and "secret", for now I have hard-coded that into my test cases.

Reply:
I recommend against hardcoding configurable values that might change in the future. Ultimately that decision rests with the developer making the change.